collection_requirements.rst

```diff
@@ -262,11 +262,18 @@ ways:
                <https://opensource.org/licenses/BSD-2-Clause>`_ license to make it possible for
                third-party modules which are licensed incompatibly with the GPLv3 to use them.
                Please consider this use case when licensing your own ``module_utils``.
-:All other code: All other code must be under the `GPL-3.0-or-later
-                 <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_.  These plugins are run
-                 inside of the Ansible controller process which is licensed under the GPLv3+ and
-                 often must import code from the controller.  For these reasons, the GPLv3+ must be
-                 used.
+:All other code in ``plugins/``: All other code in ``plugins/`` must be under the `GPL-3.0-or-later
+                                 <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_.  These plugins
+                                 are run inside of the Ansible controller process which is licensed under
+                                 the ``GPL-3.0-or-later`` and often must import code from the controller.
+                                 For these reasons, the GPLv3+ must be used.
```

Comment:
`s/GPLv3+/GPL-3.0-or-later/`

Reply:
Implemented in 48da08635ce9bdf7a571572610f17f3dbaae446b.